configs/sample_yamls/serviceops/sample_service_config.yml

```diff
@@ -0,0 +1,172 @@
+#
+########################## types ########################
+#
+types:
```

Comment:
Captured ad issue https://github.com/istio/mixer/issues/802

Thanks Louis for adding the issue.